tests/popup-test.js

```diff
@@ -71,20 +74,10 @@ describe('addToDict(tempScript, htmlString, foundThis, key, category)', function
   });
 });
 
-describe('apps.json should be valid json', function () {
-
-  var fileIsOk;
-
-  beforeEach(() => {
-    sinon.stub(window, 'fetch');
-  });
-
-  afterEach(() => {
-    window.fetch.restore();
-  });
-
-
-  it('should return valid JSON', function () {
-    expect(1).to.equal(true);
+describe('vendors.json should be valid json', function () {
+  it('should say that the file exists', function () {
```

Comment:
I'm having a bit of trouble with this test. Trailing commas will break the chrome extension, but I have been unable to find a way to check that they exist in the vendors.json file. I have tried using JSON.stringify, but it did not work because it ignores trailing commas, even though trailing commas break the chrome extension. I have also tried to use JSON.parse but it didn't work because the parameter has to be a string. Open to your suggestions on this @choumx